packages/suite/src/actions/wallet/coinjoinClientActions.ts

```diff
@@ -5,14 +5,16 @@ import {
     SerializedCoinjoinRound,
     CoinjoinRequestEvent,
     CoinjoinResponseEvent,
+    ClientEvents,
 } from '@trezor/coinjoin';
 import { arrayDistinct } from '@trezor/utils';
+import { RoundPhase } from '@trezor/coinjoin/src/enums';
```

Comment:
why this change? RoundPhase is exported from `@wallet-types/coinjoin`

Reply:
done